### PR TITLE
Revert "Revert "Schema registry <> Device registry""

### PR DESCRIPTION
**Pull request by NoahNsimbe**
Reverts airqo-platform/AirQo-api#455

### CODE REVIEW

File: kafka/clusters/prod-kafka-cluster.yaml [hunk 1]
```diff
@@ -1,12 +1,18 @@
-apiVersion: kafka.strimzi.io/v1beta1
```
Comment (Baalmart) [inline kafka/clusters/prod-kafka-cluster.yaml:1]: @NoahNsimbe 
A quick one, could we have these k8s deployment files under the k8s directory.....just to maintain the original flow of things? @danielogen

**Review by Baalmart — CHANGES_REQUESTED** [review]
just realised something that might require some changes.....awaiting thoughts/concerns...please see the recent comment about the location of the deployment files. @danielogen

File: kafka/clusters/prod-kafka-cluster.yaml (hunk 1 above)
Comment (Baalmart) [reply kafka/clusters/prod-kafka-cluster.yaml:1]: I guess this comment was intended to the corresponding PR which is a work in progress.......let me migrate it there shortly......I am of the view that we do not revert but rather concentrate on using the other corresponding PR to bring in new changes accordingly. Thanks.

Comment (NoahNsimbe) [reply kafka/clusters/prod-kafka-cluster.yaml:1]: thanks @Baalmart , the requested changes have been worked on in #462

**Comment by NoahNsimbe** [comment]
This pull request has been substituted with #462 and #458

**Pull request closed without merging**